linchpin/provision/roles/libvirt/tasks/teardown_libvirt_node.yml

```diff
@@ -2,6 +2,62 @@
       res_count: '{{ (res_count | default([])) + [item | int] }}'
   with_sequence: "start=0 end={{ res_def['count'] | default(1) |int - 1 }}"
 
+- block:
+  - name: "halt node"
+    virt:
+      name: "{{ libvirt_resource_name }}"
+      state: destroyed
+      uri: "{{ instance[1]['uri'] | default('qemu:///system') }}"
+    with_nested:
+      - ["{{ res_def }}"]
+      - "{{ res_count }}"
+      - ["{{ res_def['name_separator'] | default('_')  }}"]
+    loop_control:
+      loop_var: instance
+    register: res_def_output
+    ignore_errors: yes
```

Comment:
`ignore_errors` is like running everything as root, it could hide problems in the process.